testsuite/microprofile-tcks/opentracing/src/test/java/org/eclipse/microprofile/opentracing/wfswarm/DeploymentProcessor.java

```diff
@@ -41,6 +41,6 @@ public void process(Archive<?> archive, TestClass testClass) {
 
     WebArchive war = WebArchive.class.cast(archive);
     war.addAsLibraries(extensionsJar);
-
+    war.setWebXML("web.xml");
```

Comment:
Hm, the only problem here is that if a TCK test bundles its own `web.xml` it will be overwritten.

Reply:
Possibly, but setting `web-fragment.xml` in WEB-INF didn't trigger it